wao/wao_strategy_controller.py

```diff
@@ -3,17 +3,15 @@
 from wao.brain_config import BrainConfig
 from wao.brain_util import create_watchers
 from wao.notifier import send_start_deliminator_message
-import sys
-import os
 
-sys.path.append(BrainConfig.EXECUTION_PATH)
-from config import Config
+from execution.config import Config
 
 
 class WAOStrategyController:
 
     def __init__(self, brain, time_out_hours, dup):
         self.brain = brain
+        BrainConfig.BRAIN = brain
```

Comment:
set this in strategy files and remove self.brain from strategies and WaoStrategyController

remove all references and parameters

wherever it's used at the end, replace by BrainConfig.BRAIN